source/_integrations/androidtv_remote.markdown

```diff
@@ -44,6 +43,132 @@ Examples of URLs to pass as activity for some popular apps:
 | Prime Video | https://app.primevideo.com
 | Disney+ | https://www.disneyplus.com
 
+Examples:
+
+```yaml
+# Launch the Netflix app
+service: media_player.play_media
+data:
+  media_content_type: url
+  media_content_id: https://www.netflix.com/title
+target:
+  entity_id: media_player.living_room_tv
+```
+
+```yaml
+# Open a specific YouTube video:
+service: media_player.play_media
+data:
+  media_content_type: url
+  media_content_id: https://www.youtube.com/watch?v=dQw4w9WgXcQ
+target:
+  entity_id: media_player.living_room_tv
+```
+
+### Switch channels
+
+You can pass the channel number to switch the channel. The channel number must be an integer.
+
+Example:
+
+```yaml
+# Change channel to number 15:
+service: media_player.play_media
+data:
+  media_content_type: channel
+  media_content_id: 15
+target:
+  entity_id: media_player.living_room_tv
+```
+
+## Remote
+
+The remote allows you to send key commands to your Android TV device with the `remote.send_command` service.
+The entity has the `current_activity` attribute that shows the current foreground app on the Android TV.
+
+{% details "List of the most common commands" %}
+
+Navigation:
+- DPAD_UP
+- DPAD_DOWN
+- DPAD_LEFT
+- DPAD_RIGHT
```

Comment:
This is unrelated? Why was this changed in this PR?

Reply:
I was changing the structure of the documentation to make the docs common to both the media player and remote platforms, and I forgot to mention that I moved the list of commands from an external link into the documentation. Sorry for that. In any case, this should have been changed considering your next comment.